pkg/syncer/reconcile/fight/logger.go

```diff
@@ -21,28 +21,28 @@ import (
 	"kpt.dev/configsync/pkg/status"
 )
 
-// fightLogger is used to log errors about fights from fightDetector at most
+// Logger is used to log errors about fights from Detector at most
 // once every 60 seconds. It has similar performance characteristics as
-// fightDetector.
+// Detector.
 //
-// Instantiate with newFightLogger().
-type fightLogger struct {
-	// lastLogged is when fightLogger last logged about a given API resource.
+// Instantiate with NewLogger().
+type Logger struct {
```

Comment:
Should this be private? I assume it's only intended to be used within this package

Reply:
Well, it is instantiated in `pkg/syncer/reconcile/apply.go`. Maybe I should just move the instantiation into the detector.

Reply:
Done